### DISCUSSION TITLE
I can't put a fixation cross within a randomized code. Can you help me?

### DISCUSSION THREAD

**Discussion by Ricardo-Piccolo** [jspsych/jsPsych#3030] [General]
I'm trying to put a fixation cross (+) in the middle of each stimulus. Those stimuli have to be randomized, but the fixation cross does not. How can I write a code like that? (e.g. cross, randomized stimulus cross, randomized stimulus, cross... and so on). Here's the current code (a part of it):

var test1_trials = { 
        timeline: [test1],
        timeline_variables: [
                {stimulus: 'cenoura.jpeg'},
		        {stimulus: 'cenoura.jpeg'},
		        {stimulus: 'coelho.jpeg'},
		        {stimulus: 'coelho.jpeg'},
		        {stimulus: 'coelho.jpeg'},
                {stimulus: 'coelho.jpeg'},
                {stimulus: 'coelho.jpeg'},
                {stimulus: 'coelho.jpeg'},
                {stimulus: 'coelho.jpeg'},
                {stimulus: 'coelho.jpeg'},
        ],
        randomize_order: true
        };
        timeline.push(test1_trials);

**Comment by jodeleeuw** [comment]
Hi @Ricardo-Piccolo,

This is covered in [step 6 of the tutorial](https://www.jspsych.org/7.3/tutorials/rt-task/#part-6-timeline-variables). Basically you just add the fixation trial to the timeline of `test1_trials`:

```js
var test1_trials = {
	timeline: [fixation, test1],
	...
}
```

**Reply by Ricardo-Piccolo** [reply]
Thanks for your time! I did this but still it is not adding the '+'. 

var test1 = {
        type: 'image-keyboard-response',
        choices: ' ',
        stimulus: jsPsych.timelineVariable('stimulus'),
        post_trial_gap: 300,
        response_ends_trial: false,
        trial_duration: 1300,
        };

        var test1_trials = { 
        timeline: ['+', test1],
        timeline_variables: [
                {stimulus: 'cenoura.png'},
		        {stimulus: 'cenoura.png'},
		        {stimulus: 'coelho.png'},
		        {stimulus: 'coelho.png'},
		        {stimulus: 'coelho.png'},
                {stimulus: 'coelho.png'},
                {stimulus: 'coelho.png'},
                {stimulus: 'coelho.png'},
                {stimulus: 'coelho.png'},
                {stimulus: 'coelho.png'},
        ],
        randomize_order: true
        };
        timeline.push(test1_trials);

**Reply by jodeleeuw** [reply]
Please check out the tutorial. It covers exactly the steps that are needed to implement it.
